pkg/k8s/cnp.go

```diff
@@ -262,6 +263,10 @@ func (c *CNPStatusUpdateContext) update(cnp *cilium_v2.CiliumNetworkPolicy, enfo
 		err   error
 	)
 
+	// Ignore LastAppliedConfigAnnotation as it can be really costly to upload
+	// this as part of the status.
+	delete(annotations, v1.LastAppliedConfigAnnotation)
```

Comment:
Does this actually work when we have to use `Update()` instead of Patch? Won't it remove the annotation from the original in the apiserver?

Reply:
Very good point. It will